diff -Nuar xenserver-core/SPECS/xenserver-install-wizard.spec xenserver-core.new/SPECS/xenserver-install-wizard.spec
--- xenserver-core/SPECS/xenserver-install-wizard.spec	2013-12-10 16:47:38.748000179 -0600
+++ xenserver-core.new/SPECS/xenserver-install-wizard.spec	2013-12-10 16:46:23.341000088 -0600
@@ -8,6 +8,8 @@
 License: LGPL+linking exception
 URL:  http://github.com/djs55/xenserver-install-wizard
 Source0: https://github.com/djs55/%{name}/archive/%{version}/%{name}-%{version}.tar.gz
+Patch0: xenserver-install-wizard-0.2.28.patch
+Patch2: xenserver-install-wizard-0.2.28.patch2
 BuildRoot: %{_tmppath}/%{name}-%{version}-root
 Requires: python newt xapi-python-devel python-argparse
 
@@ -16,6 +18,8 @@
 
 %prep 
 %setup -q
+%patch0 -p1
+%patch2 -p1
 
 %build
 
