diff -Naur xenserver-install-wizard-0.2.28/networkscripts.py xenserver-install-wizard-0.2.28mod/networkscripts.py
--- xenserver-install-wizard-0.2.28/networkscripts.py	2013-12-04 07:37:12.000000000 -0600
+++ xenserver-install-wizard-0.2.28mod/networkscripts.py	2013-12-05 10:28:36.870000194 -0600
@@ -57,12 +57,17 @@
 
 			if "BOOTPROTO" in sysconfig and sysconfig["BOOTPROTO"] == "dhcp":
 				interfaces_to_reconfigure[d] = ("DHCP", "", "", "", "")
-			if "BOOTPROTO" in sysconfig and sysconfig["BOOTPROTO"] == "none":
-				if "IPADDR" in sysconfig and "NETWORK" in sysconfig and "NETMASK" in sysconfig:
+			if "BOOTPROTO" in sysconfig and (sysconfig["BOOTPROTO"] == "none" or sysconfig["BOOTPROTO"] == "static"):
+				if "IPADDR" in sysconfig and "NETMASK" in sysconfig:
 					ipaddr = sysconfig["IPADDR"]
-					network = sysconfig["NETWORK"]
+                                        if "GATEWAY" in sysconfig:
+                                                gateway = sysconfig["GATEWAY"]
+                                        elif "NETWORK" in sysconfig:
+                                                gateway = sysconfig["NETWORK"]
+                                        else:
+                                                gateway = ""
 					netmask = sysconfig["NETMASK"]
-					interfaces_to_reconfigure[d] = ("Static", ipaddr, netmask, network, "")
+					interfaces_to_reconfigure[d] = ("Static", ipaddr, netmask, gateway, "")
 		return (file_changes, interfaces_to_reconfigure)
 	finally:
 		x.logout()
